languages/translations/en.json

```diff
@@ -254,7 +254,7 @@
                 "name": "Take the ramp on the right onto {way_name}",
                 "destination": "Take the ramp on the right towards {destination}",
                 "exit": "Take exit {exit}",
-                "exit_destination": "Take exit {exit} towards {destination}"
+                "exit_destination": "Take exit {exit} on the right towards {destination}"
```

Comment:
"Fixed" this by removing the left/right indication in https://github.com/Project-OSRM/osrm-text-instructions/pull/124.